src/components/TextField/TextField.tsx

```diff
@@ -410,6 +410,12 @@ export function TextField({
     'aria-multiline': normalizeAriaMultiline(multiline),
   });
 
+  const textFieldStyles = classNames(
+    styles.Backdrop,
+    connectedLeft && styles.BackdropWithConnectedLeft,
+    connectedRight && styles.BackdropWithConnectedRight,
```

Comment:
Could we do 
```suggestion
    newDesignLanguage && connectedRight && styles.BackdropWithConnectedRight,
```
instead?